css/faq.css

```diff
@@ -0,0 +1,73 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+.faq {
+  background: rgba(0, 0, 0, 0.6);
+}
+
+.questions {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-evenly;
+}
+
+h3 {
```

Comment:
Can you specify the element's parent (e.g.  .faq>h3 )? 
There might be multiple h3 elements on the page, and we want the style for only some.